teacher_class.rb

```diff
@@ -0,0 +1,12 @@
+require './Person'
+
+class Teacher < Person
+  def initialize(age, name, specialization)
+    super(age, name)
+    @specialization = specialization
```

Comment:
- [ ] in order for this initialize method to work please make sure to set the default values for age and parent_permission as well.